frontend/ui/elements/reader_menu_order.lua

```diff
@@ -23,7 +23,6 @@ local order = {
         "highlight_options",
         "change_font",
         "hyphenation",
-        "read_timer",
```

Comment:
Firstly, for consistent reason. It's placed under tools tab in filemanager. We should either keep it consistent between apps or only enable it in reader.

Secondly, that tab is named typeset, which means anything that controls rendering results. A timer doesn't fit into this category. With the new UX proposed by @baskerville and the new separator feature, we can combine navigation and typeset into one read menu, then it makes more sense to put timer there.